src/migrateDashboard.ts

```diff
@@ -92,9 +94,21 @@ export function migrateDashboard(
   body.pages.forEach((legacyPage: LegacyDashboardPage, index: number) => {
     const pageKey = `p-${index}`;
     const content: DashboardPageState<"serialized">["content"] = {};
+    keysOfDisconnectedWidgets[pageKey] = [];
     legacyPage.content.forEach((widget) => {
       const leafKey = widget.key;
       const widgetPluginKey = _getLegacyWidgetPluginKey(widget.bookmark);
+
+      if (
+        widget.bookmark.value.body?.configuration?.tabular?.isConnected ===
```

Comment:
I suspect that `isConnected` might have been set in this place for table widgets but probably not for other types of widgets. We should check whether this information always lives there or not.